### PR TITLE
feat(exporter/windows): expose physical_disk collector

### LINKED ISSUE #5882: Expose new collectors/config from windows_exporter to flow
### Request

Now that we are using the upstream version of windows_exporter we should review all the new collectors added and expose those to flow. 

### Use case

Use new windows_exporter features.

### PR DESCRIPTION
**Pull request by hainenber**
#### Which issue(s) this PR fixes

Fixes #5882 

#### Notes to the Reviewer
The new collector is set as default by`windows_exporter`. Should we follow the course or keep it as non-default to keep it simpler?

#### PR Checklist

- [x] CHANGELOG.md updated
- [x] Documentation added
- [x] Tests updated
- [x] Config converters updated

### CODE REVIEW

**Comment by mattdurham** [comment]
We should decide a policy going forward on it. Technically we have said that changing metrics is not a breaking change. Do we have a feel for how many metrics it creates?

**Comment by hainenber** [comment]
There are [11 new metrics](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.physical_disk.md#metrics) if this is enabled by default.

File: docs/sources/static/configuration/integrations/windows-exporter-config.md [hunk 1]
```diff
@@ -114,6 +114,16 @@ Full reference of options:
     # Maps to collector.service.services-where in windows_exporter
     [where_clause: <string> | default=""]
 
+  # Configuration for physical disk on Windows 
+  physical_disk:
+    # Regexp of volumes to include. Disk name must both match include and not match exclude to be included.
+    # Maps to collector.logical_disk.disk-include in windows_exporter
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/integrations/windows-exporter-config.md:120]:
```suggestion
    # Maps to collector.logical_disk.disk-include in windows_exporter.
```

File: docs/sources/static/configuration/integrations/windows-exporter-config.md [hunk 2]
```diff
@@ -114,6 +114,16 @@ Full reference of options:
...
+    [include: <string> | default=".+"]
+
+    # Regexp of volumes to exclude. Disk name must both match include and not match exclude to be included.
+    # Maps to collector.logical_disk.disk-exclude in windows_exporter
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/integrations/windows-exporter-config.md:124]:
```suggestion
    # Maps to collector.logical_disk.disk-exclude in windows_exporter.
```

**Review by clayton-cornell — COMMENTED** [review]
Minor doc suggestion

**Comment by mattdurham** [comment]
Looks good enabled automerge once conflicts are resolved.

**Review by mattdurham — APPROVED** [review]
LGTM

**Review by clayton-cornell — APPROVED** [review]

File: pkg/integrations/windows_exporter/config_windows.go [hunk 3]
```diff
@@ -87,6 +90,10 @@ var DefaultConfig = Config{
 		Include:   collector.ConfigDefaults.Net.NicInclude,
 		Exclude:   collector.ConfigDefaults.Net.NicExclude,
 	},
+	PhysicalDisk: PhysicalDiskConfig{
+		Include: collector.ConfigDefaults.PhysicalDisk.DiskInclude
```
Comment (mattdurham) [inline pkg/integrations/windows_exporter/config_windows.go:94]: Missing a comma here

**Review by mattdurham — CHANGES_REQUESTED** [review]
Need a comma

**Review by mattdurham — APPROVED** [review]
lets check now

**Pull request merged**